### PR TITLE
Update hint to prevent crash on date x values

### PR DESCRIPTION
**Pull request by Domino987**
With this Change, the Hint does not crash, if a date is provided as x value

### CODE REVIEW

File: src/plot/hint.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const ORIENTATION = {
  */
 function defaultFormat(value) {
   return Object.keys(value).map(function getProp(key) {
-    return {title: key, value: value[key]};
+    return { title: key, value: Object.prototype.toString.call(value[key]) === '[object Date]' ? value[key].toDateString() : value[key] };
```
Comment (mcnuttandrew) [inline src/plot/hint.js]: maybe break this out into a function and use typeof instead?

Comment (Domino987) [reply src/plot/hint.js]: I will look into it to change it. What ist your linting for the tests?

Comment (mcnuttandrew) [reply src/plot/hint.js]: I'm not sure I understand your question? We use eslint, you can run it locally by running 

```sh
npm run lint
```

Comment (Domino987) [reply src/plot/hint.js]: This throws 24237 erros:  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style.
I thought that i have to change my linting to ignore those for example.

Comment (Domino987) [reply src/plot/hint.js]: Seems like Windows is the problem. I will work on it and try to resolve it.

**Comment by mcnuttandrew** [comment]
You should make sure to squash your commits to address the CLA issue above

**Comment by Domino987** [comment]
Ok I will. My linter is pretty strict (everything alphabetical ordered and such) so that makes testing difficult, but I will change it.

**Comment by Domino987** [comment]
See #1077. Cleaned it up and reopened

**Pull request closed without merging**